webf/lib/src/dom/element.dart

```diff
@@ -1002,6 +1002,11 @@ abstract class Element extends Node with ElementBase, ElementEventMixin, Element
 
   @mustCallSuper
   void setAttribute(String qualifiedName, String value) {
+    if (value.startsWith(';')) value = value.substring(1);
+    if (HIDDEN == qualifiedName) {
```

Comment:
Hidden attribute have three different kind of values. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

<img width="278" alt="image" src="https://user-images.githubusercontent.com/4409743/200093347-532a9813-5f92-4dc2-812e-e32b06e34883.png">

Reply:
thx,  it really not comprehensive, I try to handle it.